ui/elements/IconGroup/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from './IconGroup';
+export type { IconGroupProps, StyledIconGroupProps } from './types';
```

Comment:
need not export StyledIconGroupProps from this barrel file
```suggestion
export type { IconGroupProps } from './types';
```